### PR TITLE
Cleaner hub logs

### PR DESCRIPTION
**Pull request by bohendo**
## Description

Changed some info logs into debug & vice versa, turned off the debug-level logs kinda bluntly, couldn't figure out how to access the config.

I didn't turn off anything important did I? Anything else you think we should add?

See bottom for sample.

## Motivation and Context

Spent a lot of time looking at logs today while debugging, made them a little prettier as I went. Or so I think.

## How Has This Been Tested?

Card's e2e tests pass + CI (hoopefully)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

```
info: Setting up API service at /"thread".
info: Setting up API service at /"payments".
info: Setting up API service at /"coinpayments".
info: Starting exchange rate polling service.
info: Starting gas estimate service.
info: Listening on port 8080.
info: Starting OnchainTransactionService...
info: Starting CoinPaymentsDepositPollingService...
info: "::ffff:10.0.18.58" "GET" "/config" "0 bytes" -> 200 ("304 bytes"; "0.012ms")
info: "::ffff:10.0.18.58" "GET" "/config" "0 bytes" -> 200 ("304 bytes"; "0.002ms")
info: "::ffff:10.0.18.58" "GET" "/config" "0 bytes" -> 200 ("304 bytes"; "0.005ms")
info: "::ffff:10.0.18.58" "GET" "/config" "0 bytes" -> 200 ("304 bytes"; "0.004ms")
info: No session found. Returning unsuccessful auth status.
info: "::ffff:10.0.18.58" "GET" "/auth/status" "0 bytes" -> 200 ("17 bytes"; "0.003ms")
info: Sending challenge nonce.
info: "::ffff:10.0.18.58" "POST" "/auth/challenge" "2 bytes" -> 200 ("48 bytes"; "0.068ms")
Hash sent to extract 0x9db7daebdcb7dff57b136ac466ea6ae22155ae574b442ecd2aee8b5f0997d055
info: "::ffff:10.0.18.58" "POST" "/auth/response" "271 bytes" -> 200 ("44 bytes"; "0.007ms")
info: "::ffff:10.0.18.58" "GET" "/channel/0x91ada9349412b999bd63e647af687417eb66457f/latest-update" "0 bytes" -> 404 ("9 bytes"; "0.009ms")
info: "::ffff:10.0.18.58" "GET" "/exchangeRate" "0 bytes" -> 200 ("78 bytes"; "0.004ms")
channel:  {
  "id": null,
  "status": "CS_OPEN",
  "user": "0x91ada9349412b999bd63e647af687417eb66457f",
  "lastUpdateOn": "1970-01-01T00:00:00.000Z",
  "state": {
    "contractAddress": "0x345ca3e014aaf5dca488057592ee47305d9b3e10",
    "user": "0x91ada9349412b999bd63e647af687417eb66457f",
    "recipient": "0x91ada9349412b999bd63e647af687417eb66457f",
    "balanceWeiHub": "0",
    "balanceWeiUser": "0",
    "balanceTokenHub": "0",
    "balanceTokenUser": "0",
    "pendingDepositWeiHub": "0",
    "pendingDepositWeiUser": "0",
    "pendingDepositTokenHub": "0",
    "pendingDepositTokenUser": "0",
    "pendingWithdrawalWeiHub": "0",
    "pendingWithdrawalWeiUser": "0",
    "pendingWithdrawalTokenHub": "0",
    "pendingWithdrawalTokenUser": "0",
    "threadCount": 0,
    "threadRoot": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "timeout": 0,
    "txCountChain": 0,
    "txCountGlobal": 0,
    "sigHub": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
    "sigUser": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
  }
}
CHANNEL UPDATE RESULT: []
THREAD UPDATE RESULT: []
info: "::ffff:10.0.18.58" "GET" "/channel/0x91ada9349412b999bd63e647af687417eb66457f/sync?lastChanTx=0&lastThreadUpdateId=0" "0 bytes" -> 200 ("33 bytes"; "0.016ms")
info: "::ffff:10.0.18.58" "GET" "/gasPrice/estimate" "0 bytes" -> 200 ("14 bytes"; "0.005ms")
info: "::ffff:10.0.18.58" "GET" "/config" "0 bytes" -> 200 ("304 bytes"; "0.005ms")
info: "::ffff:10.0.18.58" "GET" "/config" "0 bytes" -> 200 ("304 bytes"; "0.004ms")
info: No session found. Returning unsuccessful auth status.
info: "::ffff:10.0.18.58" "GET" "/auth/status" "0 bytes" -> 200 ("17 bytes"; "0.002ms")
info: Sending challenge nonce.
info: "::ffff:10.0.18.58" "POST" "/auth/challenge" "2 bytes" -> 200 ("48 bytes"; "0.002ms")
Hash sent to extract 0x27246d79f42a30c6657f09d378cbe6d6084e2b12870280ebcf9cbe2e3947cb9f
info: "::ffff:10.0.18.58" "POST" "/auth/response" "271 bytes" -> 200 ("44 bytes"; "0.003ms")
info: "::ffff:10.0.18.58" "GET" "/channel/0x26242f3485256e9be2873e7426116ce5019d9cdd/latest-update" "0 bytes" -> 404 ("9 bytes"; "0.005ms")
info: "::ffff:10.0.18.58" "GET" "/exchangeRate" "0 bytes" -> 200 ("78 bytes"; "0.004ms")
channel:  {
  "id": null,
  "status": "CS_OPEN",
```

### CODE REVIEW

File: modules/hub/src/util/logging.ts [hunk 1]
```diff
@@ -93,7 +96,7 @@ export class SCLogger {
 
     const c = console as any
 
-    if (process.env.NODE_ENV !== 'production' || !this.logdna) {
+    if ((process.env.NODE_ENV !== 'production' || !this.logdna) && levelno >= logLevelLimit) {
```
Comment (bohendo) [inline modules/hub/src/util/logging.ts]: Seems a little clunky to just compare to a hardcoded limit, not sure how to use hub's config. Maybe just use process.env?

File: modules/hub/src/ChannelsService.ts [hunk 2]
```diff
@@ -586,8 +586,8 @@ export default class ChannelsService {
 
     console.log('USER:', user)
```
Comment (rhlsthrm) [inline modules/hub/src/ChannelsService.ts]: Let's also make these LOG.debug?

File: modules/hub/src/util/logging.ts (hunk 1 above)
Comment (rhlsthrm) [reply modules/hub/src/util/logging.ts]: process.env is probably fine for this.

Comment (elenko5) [reply modules/hub/src/util/logging.ts]: Yes

**Comment by bohendo** [comment]
Update:
 - Set the log level via process.env (but still default to 20 if none is provided)
 - Replaced remaining console.logs with LOG.info or LOG.debug

**Pull request merged**